cluster/examples/workloads/kubernetes/wordpress-gcp/workload.yaml

```diff
@@ -1,15 +1,3 @@
-apiVersion: compute.crossplane.io/v1alpha1
```

Comment:
i think we can keep the name of this file to be app.yaml, i think i like that naming convention since what this file is really capturing is a KubernetesApplication type.

Reply:
Will do.  I reverted the name to fit the instructions.  On my first pass I kept the app.yaml name and revised the docs to match.